src/common/exif.cc

```diff
@@ -1110,6 +1120,16 @@ int dt_exif_write_blob(uint8_t *blob, uint32_t size, const char *path, const int
       dt_remove_exif_keys(imgExifData, keys, n_keys);
     }
 
+    // remove subimage* trees, related to thumbnails or HDR usually
+    for(Exiv2::ExifData::iterator i = imgExifData.begin(); i != imgExifData.end();)
+    {
+      std::string needle = "Exif.SubImage";
+      if(i->key().compare(0, needle.length(), needle) == 0)
```

Comment:
would not these information be useful to read hdr DNG files? (I was planning to test those at some point, when my camera gets back from service)